aad/Ballerina.toml

```diff
@@ -1,7 +1,11 @@
-[project]
-org-name= "ballerinax"
-version= "0.9.0"
-license= ["Apache-2.0"]
+[package]
+org= "ballerinax"
+name= "azure.ad"
+version= "1.0.0"
```

Comment:
Connector version should be in SNAPSHOT in the master branch

Reply:
Fixed